docs/source/quick_start.rst

```diff
@@ -4,83 +4,171 @@ Quick Start
 This guide is intended to jump start new Kokkos users (and beginners, in particular).
 
 
-Download Latest and Build 
------------------------------
+Prerequisites
+~~~~~~~~~~~~~
 
-.. note::
+To complete this tutorial, you'll need:
+
+* a compatible operating system (e.g. Linux, macOS, Windows).
+* a compatible C++ compiler that supports at least C++17.
+* `CMake <https://cmake.org/>`_ and a compatible build tool for building the
```

Comment:
maybe spell out the minimum version here? we probably require one implicitly ... but might be outside of the scope here